### ISSUE TITLE
Add support for Chia (XCH)

### ISSUE THREAD

**Issue by matthewdowns** [LedgerHQ/ledger-live-common#1185]
No worries if put on the backburner, this coin is so new. Just putting it out there that myself and many other people would love Chia support on Ledger.

**Comment by martinsthilaire** [comment]
I would have expected Ledger to be more on the ball than that to be honest, the chia wallets are following pretty standard conventions. Pools are forming soon, please keep us safe haha ^^ Thanks for your good work!

**Comment by lpreimesberger** [comment]
tossing in too - there's no 'safe' wallet with chia support right now and it's a pretty major currency based on price/market cap.

**Comment by Haxiboy** [comment]
I'd like to see chia in ledger too.

**Comment by pablopgus** [comment]
Stepping in to support this feature. Chia rocks!

**Comment by BravoRotor2341** [comment]
All coins with high market price must be supported by Ledger.

**Comment by mist0706** [comment]
Just got my first chia, When is the ledger support coming?

**Comment by deanhadzi** [comment]
Please support chia on your platform! Thanks in advance Ledger.

**Comment by SuuKool** [comment]
Please, this needs to be done!

**Comment by mmoadvert** [comment]
make chia support in ledger live

**Comment by joeharrison714** [comment]
+1 for Chia

**Comment by DCYeahThatsMe** [comment]
+1 absolutely would love to have my Nano S support CHIA! It would even be reason to buy another or upgrade to the X!

**Comment by ndsutter** [comment]
+1 been farming for a while and have a couple Chia, would love for Ledger to support!

**Comment by hajes** [comment]
support would be nice but no developer would bother by some unknown coin that is not even listed on major exchanges. Coin that may cease to exist with a half year. No commercial company would take such a risk ;-)

**Comment by ndsutter** [comment]
> support would be nice but no developer would bother by some unknown coin that is not even listed on major exchanges. Coin that may cease to exist with a half year. No commercial company would take such a risk because it is waste of company resources ;-)

...founded and run by the guy who invented BitTorrent, trading volume and market cap way above a lot of other coins that are supported.  Hardly "unknown coin" and the idea that it will cease to exist is silly.

**Comment by Mainnia** [comment]
Pls add chia we have no safe wallet!

**Comment by Phoenix603** [comment]
+1

**Comment by baronduke** [comment]
+1

**Comment by tydeno** [comment]
+1

**Comment by bkromeo** [comment]
Please add chia (XCH) to leger

**Comment by ligams** [comment]
+1

**Comment by schnub69** [comment]
+1

**Comment by elisir1986** [comment]
I think it is probably just a matter of time for this one with you guys... but please hurry, we need all our coins in the drawer so we sleep better at night. :)

**Comment by webgkv** [comment]
+ 1

**Comment by webgkv** [comment]
+1

**Comment by snoopsbne** [comment]
+1

**Comment by alexverbitsky** [comment]
+1

**Comment by Valentin-Miroshnichenko** [comment]
+1

**Comment by bitum7** [comment]
Please add chia (XCH) to leger

**Comment by horego** [comment]
XCH is worth it adding to ledger :) Please add it

**Comment by maxXx1** [comment]
+1 for chia

**Comment by bulaten** [comment]
+1 that would be great!

**Comment by xerces13** [comment]
+1

**Comment by pablopgus** [comment]
+1

**Comment by 0x13ens** [comment]
+1 can we get xch please?

**Comment by WDebel** [comment]
+1

**Comment by delucca** [comment]
+1

**Comment by abranski** [comment]
+1

**Comment by greenhawk89** [comment]
+1

**Comment by derp55555** [comment]
Add XCH please.

**Comment by zadimx** [comment]
+1

**Comment by sykmer** [comment]
+1

**Comment by feurer98** [comment]
+1

**Comment by ak1ca1** [comment]
+1

**Comment by matthewdowns** [comment]
Closing since support for this on Ledger devices first needs to be developed by Chia. https://github.com/Chia-Network

**Comment by joeharrison714** [comment]
@matthewdowns Chia is open source, don't see why you would need to wait on them for something to be developed

**Comment by hoffmang9** [comment]
Ledger does not yet support BLS in G2 so this frankly should be open until you do.

**Comment by sargonas** [comment]
> Closing since support for this on Ledger devices first needs to be developed by Chia. https://github.com/Chia-Network

@matthewdowns  Any chance we could get this re-opened, since it's _actually_ dependent on proper BLS support on the Ledger side, not Chia's? (Seems like just a simple misunderstanding... it happens)

**Comment by derp55555** [comment]
> > Closing since support for this on Ledger devices first needs to be developed by Chia. https://github.com/Chia-Network
> 
> @matthewdowns Any chance we could get this re-opened, since it's _actually_ dependent on proper BLS support on the Ledger side, not Chia's? (Seems like just a simple misunderstanding... it happens)

Please look into this @matthewdowns

**Comment by andresionek91** [comment]
Support for XCH please

**Comment by chriseomi** [comment]
Would love to see XCH supported :)

**Comment by derp55555** [comment]
Add Chia (XCH) support please!!

**Comment by chriseomi** [comment]
Would love to see XCH supported

**Comment by glxhb** [comment]
+ 1

**Comment by darvd29** [comment]
I will buy a Trezor device immediately if you add support for Chia :)

**Comment by delucca** [comment]
IMHO we should create a new issue, since they closed it

**Comment by serfriz** [comment]
> Ledger does not yet support BLS in G2 so this frankly should be open until you do.

Any update about this?

**Comment by Malinovski-George** [comment]
Need chia support

**Comment by NovaMi** [comment]
+1

**Comment by alfondotnet** [comment]
wen chia support
